build/Targets/Imports.targets

```diff
@@ -118,17 +118,16 @@
           DestinationFolder="$(OutDir)" />
   </Target>
 
-  <!-- Guard against MSBuild picking the wrong SDK version. Its resolver does not always properly enforce
-       the SDK specified in global.json 
-       
-       https://github.com/dotnet/core-setup/issues/3805
-       -->
-  <Target Name="BeforeBuild" Condition="'$(MSBuildRuntimeType)' != 'Core'">
-    <ValidateBuildEnvironment
-      MSBuildBinPath="$(MSBuildBinPath)"
-      MSBuildMinimumFileVersion="15.7.0"
-      MSBuildMinimumDisplayVersion="MSBuild 15.7"
-      />
+  <UsingTask TaskName="Microsoft.DotNet.Arcade.Sdk.CompareVersions" AssemblyFile="$(RoslynToolsBuildTasksAssembly)" />
+  <UsingTask TaskName="Microsoft.DotNet.Arcade.Sdk.SingleError" AssemblyFile="$(RoslynToolsBuildTasksAssembly)" />
```

Comment:
Why are things in Arcade with AssemblyFile == $(**Roslyn**)ToolsBuildTasksAssembly?

Reply:
Temporary hack.

Reply:
Will move these to Arcade SDK proper.